content/rc/subscriptions/maintenance/set-maintenance-windows.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: Set maintenance windows
+LinkTitle: Set maintenance windows
+description: Shows how to set manual maintenance windows and skip maintenance.
+weight: $weight
+alwaysopen: false
+toc: "true"
+headerRange: "[1-3]"
+categories: []
+aliases: 
+---
+
+By default, Redis will maintain your Redis Cloud subscriptions and databases as needed. During maintenance, you may notice some latency when connecting to your databases. For more information about the types of maintenance and how Redis maintains your subscriptions, see [Maintenance]({{<relref "/rc/subscriptions/maintenance">}}).
+
+You typically do not need to set a maintenance window manually as Redis will attempt to perform maintenance during low-traffic hours when possible. If you want to control when Redis performs maintenance for a Flexible or Annual subscription, you can [set manual maintenance windows](#set-manual-maintenance-windows) to ensure non-urgent maintenance will occur at set times. Or, you can [skip non-urgent maintenance temporarily](#skip-maintenance-temporarily) during busy periods.
```

Comment:
IMO, we don't need to distinguish between the Flexible and Annual subscriptions as both of them are Flexible.